docs/reference/getting-started.asciidoc

```diff
@@ -322,10 +298,8 @@ Now that we've gotten a glimpse of the basics, let's try to work on a more reali
 --------------------------------------------------
 // NOTCONSOLE
 
-For the curious, this data was generated using http://www.json-generator.com/[`www.json-generator.com/`], so please ignore the actual values and semantics of the data as these are all randomly generated.
-
-You can download the sample dataset (accounts.json) from https://github.com/elastic/elasticsearch/blob/master/docs/src/test/resources/accounts.json?raw=true[here]. Extract it to our current directory and let's load it into our cluster as follows:
-
+. Index the account data into the _bank_ index with the following `_bulk` request.
```

Comment:
> _bank_ index

I've been using literals for index names. So this would be `bank` index.

I'm not sure I have a strong preference as long as we're consistent, although I'm not a big fan of italics generally.

Reply:
According to the style guide, it should be `bank` index